app.js

```diff
@@ -31,74 +30,16 @@ function getContractABI(contractAddress, apiKey) {
         });
 }
 
-// async function getLinkBalanceOfContract(contractAddress) {
-//     const contractABI = await getContractABI(linkContractAddress, apiKey)
-
-//     if ("abi" in contractABI) {
-
-//         const LinkTokenContract = new ethers.Contract(
-//             linkContractAddress,
-//             contractABI.abi,
-//             signer
-//         )
-
-//         return LinkTokenContract.balanceOf(contractAddress)
-//             .then(balanceHex => {
-
-//                 const balance = ethers.utils.formatEther(ethers.BigNumber.from(balanceHex._hex).toString())
-
-//                 return parseFloat(balance)
-//             })
-//             .catch(err => {
-//                 console.log('Error: ', err.toString())
-//                 return 0
-//             });;
-
-//     }
-//     else {
-//         return 0;
-//     }
-// }
-
-// async function fundContract(contractAddress, links) {
-
-//     const contractABI = await getContractABI(linkContractAddress, apiKey)
-
-//     if ("abi" in contractABI) {
-
-//         const LinkTokenContract = new ethers.Contract(
-//             linkContractAddress,
-//             contractABI.abi,
-//             signer
-//         )
-
-//         return LinkTokenContract.transfer(contractAddress, ethers.utils.parseUnits(links.toString(), 18))
-//             .then(tx => {
-
-//                 //console.log(tx.hash)
-//                 return ({ "Success": `${contractAddress} is funded with ${links} LINK token(s) successfully. The corresponding transaction hash is ${tx.hash}.` })
-//             })
-//             .catch(err => {
-//                 console.log(err.toString())
-//                 return ({ "Error": `Error occured while funding the contract. Please try again.` })
-//             });;
-
-//     }
-//     else {
-//         return ({ "Error": "Error in the Contract's ABI" })
-//     }
-// }
-
 function getEligiblePoolData() {
 
     timestampNow = Math.floor(Date.now() / 1000)
 
     timeStampBefore7Days = Math.floor(Date.now() / 1000) - (7 * 24 * 60 * 60)
 
-    console.log(timestampNow, timeStampBefore7Days)
+    // console.log(timestampNow, timeStampBefore7Days)
 
     const query = `{
-        pools(orderBy: createdAt, where:{dataProvider: "${dataProviderAddress}", expiryTime_lt: ${timestampNow}, expiryTime_gte: ${timeStampBefore7Days}, statusFinalReferenceValue: "Open", referenceAsset_starts_with: "https://", referenceAsset_contains: "ipfs", referenceAsset_ends_with: ".json"}) {
+        pools(where:{dataProvider: "${dataProviderAddress}", expiryTime_lt: ${timestampNow}, expiryTime_gte: ${timeStampBefore7Days}, statusFinalReferenceValue: "Open", referenceAsset_starts_with: "https://", referenceAsset_contains: "ipfs", referenceAsset_ends_with: ".json"}) {
```

Comment:
Can you add a comment that it was a conscious decision to not include statusFinalReferenceValue = "Challenged" to avoid resubmitting values automatically when a submission is challenged. I feel that’s pretty important, especially when others take inspiration from Code when building oracles for DIVA Protocol